support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/thankYouHeader.tsx

```diff
@@ -17,12 +18,14 @@ export const header = css`
 	}
 `;
 
-export const headerTitleText = css`
+export const headerTitleText = (
+	digitalFontSizeReduce?: boolean,
```

Comment:
Instead of turning this into a method, which creates more CSS to push down the pipes, we could just use the `style` prop in `heading.tsx`, which not only would create less css, keeps that value colocated with the relevant component.

[It's described quite well here](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).

There's a little bit of me that wonders if this style should actually live in the header component as it seems like a strange coupling as it currently stands, but it works.

Reply:
Not sure how I can get the  font to resize dependent on screen size using  this `Heading.style` method recommended? All the styles I see used in the codebase do not check the screen-size like this function does? SerializedStyles appear to be required. Have uploaded as far as I could get, however, `digital-subscriber-thank-you/thankyouHeader.tsx` line 26 is stuck in 34px regardless of screensize..

Reply:
Removed `Heading.style` as did not appear to  support media resizing.